include/deal.II/distributed/tria.h

```diff
@@ -249,7 +249,7 @@ namespace parallel
      */
     template <int dim, int spacedim = dim>
     class Triangulation
-      : public dealii::parallel::TriangulationBase<dim, spacedim>
+      : public dealii::parallel::DistributedTrinagulationBase<dim, spacedim>
```

Comment:
There's a typo in `Triangulation` here and everywhere else.

Reply:
Thanks! Ups. I have updated theses...